File: ZfCustomView/app/src/main/java/zhanf/com/zfcustomview/main/activity/SelectorTvActivity.java
package zhanf.com.zfcustomview.main.activity;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;
import android.widget.Toast;

import zhanf.com.zfcustomview.R;

public class SelectorTvActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_selector);

        findViewById(R.id.stv).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Toast.makeText(SelectorTvActivity.this, "onClick", Toast.LENGTH_SHORT).show();
            }
        });
    }
}

File: ZfCustomView/app/src/main/java/zhanf/com/zfcustomview/main/activity/SelectorTvActivity.kt
package zhanf.com.zfcustomview.main.activity

import android.os.Bundle
import android.view.View
import android.widget.Toast

import zhanf.com.zfcustomview.R

/**
 * @author sincerity
 */
class SelectorTvActivity : AbstractActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_selector)

        findViewById<View>(R.id.stv).setOnClickListener { Toast.makeText(this@SelectorTvActivity, "onClick", Toast.LENGTH_SHORT).show() }
    }
}
